vendors/liebearth/profiles/liebearth_eu_868_class_A.yaml

```
@@ -0,0 +1,89 @@
+# ID that represents the profile of your device model.
+# SHALL Follow this convention: <vendorID> (lowercase) + '_' + <Profile Group> + '_' <LoRaWAN L2 & PHY versions> + '_class' + <LoRaWAN class A, B or C>
+# Follow the information below to know the profile group to use in the ID
+#    Profile Group
+#    RFGroup1:  eu863-870, kr920-923, in865-867, ru864-870
+#    RFGroup2:  us902-928, cn470-510, au915-928 (only for LoRaWAN 1.0.2revB or below)
+#    RFGroup3:   au915-928 (starting from LoRaWAN 1.0.2revC)
+#    RFGroup4: as923
+# Please do not change the existing IDs (Some may follow the old format)
+# The name of the profile .yaml file should be the same as the following ID
+id: liebearth_eui_868_class_A
```

```suggestion
id: lbearth_RFGroup1_1.0.4_class_A
```

As mentioned in the comment above, this is the right way to identify a profile on our catalog.

After updating do not forget to update the models which use this profile.